step-api/src/main/java/org/jenkinsci/plugins/workflow/steps/AbstractSynchronousNonBlockingStepExecution.java

```diff
@@ -65,7 +65,6 @@ public void stop(Throwable cause) throws Exception {
         if (task != null) {
             task.cancel(true);
         }
-        getContext().onFailure(cause);
```

Comment:
Where is this called now? In [doTerm](https://github.com/jenkinsci/workflow-plugin/pull/141/files#diff-16ebc446122b84ab0f87d8727295f655R287)? So any `AbstractSynchronousNonBlockingStepExecution` is now virtually not cancellable without requiring a click on `doTerm` link, right?